Content.Server/Medical/Stethoscope/Components/StethoscopeComponent.cs

```diff
@@ -0,0 +1,28 @@
+using System.Threading;
+using Content.Shared.Actions.ActionTypes;
+using Robust.Shared.Utility;
+
+namespace Content.Server.Medical.Components
+{
+    /// <summary>
+    /// Adds an innate verb when equipped to use a stethoscope.
+    /// </summary>
+    [RegisterComponent]
+    public sealed class StethoscopeComponent : Component
+    {
+        public bool IsActive = false;
+
+        public CancellationTokenSource? CancelToken;
+
+        [DataField("delay")]
+        public float Delay = 2.5f;
+
+        public EntityTargetAction Action = new()
+        {
+            Icon = new SpriteSpecifier.Texture(new ResourcePath("Clothing/Neck/Misc/stethoscope.rsi/icon.png")),
+            Name = "stethoscope-verb",
+            Priority = -1,
```

Comment:
```suggestion
            Priority = -1,
            Whitelist = new() { Components = new string[] { "MobState" } },
```

Reply:
I'm planning on merging the 2 target actions soonTM and might end up changing that, (it's currently unused anywhere)